frontend/sw360-portlet/src/main/resources/content/Language.properties

```diff
@@ -870,6 +870,7 @@ project.manager=Project manager
 project.name=Project name
 project.name.first.letters=Project Name (first letters)
 project.namelinks=Project NameLinks
+project.obligation=Project Obligation
```

Comment:
There is already a key with name `project.obligation`. Same for `Language_vi.properties`

Reply:
removed duplicate